spaceship/lib/spaceship/tunes/display_family.rb

```diff
@@ -0,0 +1,66 @@
+require_relative "../module"
+
+module Spaceship
+  module Tunes
+    # A definition of differnet styled displays used by devices
+    # that App Store Connect supports storing screenshots.
+    # Display styles often only vary based on screen resolution
+    # however other aspects of a displays physical apperance are
+    # also factored (i.e if the home indicator is provided via
+    # a hardware button or a software interface).
+    class DisplayFamily
+      # The display family name from the App Store Connect API that
+      # is used to assign `Spaceship::Tunes::AppScreenshot` objects
+      # to the correct display families via its `device_type` property.
+      attr_accessor :name
+
+      # The user friendly name of this defintion as seen in the Media
+      # Manager in App Store Connect.
```

Comment:
```suggestion
      #
      # Source: Media Manager in App Store Connect.
```

line 17 also has to be edited